Classes/BEMSimpleLineGraphView.h

```diff
@@ -138,6 +138,13 @@ IB_DESIGNABLE @interface BEMSimpleLineGraphView : UIView <UIGestureRecognizerDel
 //------------------------------------------------------------------------------------//
 
 
+/// If set to YES, the graph will be constantly showing the most recent selected circle. Default value is NO.  
+@property (nonatomic) BOOL alwaysShowSelectedCircle;
```

Comment:
The name of the property could be a little bit more explicit. 
How do you feel about something like `persistLatestPopUpLabel`?